CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+# 0.65.2
+
```

Comment:
```suggestion
- No changes. This is a fixup release for sake of Packit deployment.

```

Reply:
Thanks, I'll update it :slightly_smiling_face: